docs/source/09_development/03_commands_reference.md

```diff
@@ -318,7 +318,7 @@ The following command deletes all the files related to a modular pipeline in you
 kedro pipeline delete <pipeline_name>
 ```
 
-Further information is available in the [pipeline documentation](../06_nodes_and_pipelines/03_modular_pipelines.md#pull-a-modular-pipeline).
+Further information is available in the [micro-packaging documentation](../06_nodes_and_pipelines/04_micro_packaging.md).
```

Comment:
I'm wondering if we need to do the redirect trick that @lorenabalan found to save people from the pain of the broken links. But I'll only suggest this if these are well-used pages.

Reply:
What is that? I'm not up to date